doc/plugin_agent_workloadattestor_docker.md

```diff
@@ -0,0 +1,61 @@
+# Agent plugin: WorkloadAttestor "docker"
+
+The `docker` plugin generates selectors based on docker labels for workloads calling the agent.
+It does so by retrieving the workload's container ID from its cgroup membership, then querying
+the docker daemon for the container's labels.
+
+| Configuration | Description |
+| ------------- | ----------- |
+| docker_socket_path | The location of the docker daemon socket (default: "unix:///var/run/docker.sock" on unix). |
+| docker_version | The API version of the docker daemon (default: "1.25").
+| cgroup_prefix | The cgroup prefix to look for in the cgroup entries (e.g. "/docker"). |
+| cgroup_container_index | The index within the cgroup path where the container ID should be found (default: 0). |
```

Comment:
IMO it is desirable to limit the number of knobs available to users wherever possible... otherwise, they tend to tweak things inappropriately and end up with either misconfigured or broken software, sometimes without even knowing. In that vein, I'd vote to leave `cgroup_prefix` and `cgroup_container_index` undocumented... I don't think they are very likely to change... If someone comes to us with a broken attestor we can ask them to tweak these undocumented values, or if an operator finds them in the code, then the assumption is that they know what they are doing

If we remove these configurables from the documentation, then we can also remove the `Cgroup entry` section below